app/styles/newcomponents/ilios-chart-tooltip.scss

```diff
@@ -9,16 +9,18 @@
   margin: 0;
   max-height: 150px;
```

Comment:
What if we just got rid of max height?  I'm guessing scrolling inside the tooltip is going to be a pain.